test/inductor/test_pattern_matcher.py

```diff
@@ -1286,6 +1286,20 @@ def repl(inp, x1, x2):
                 # addmm should be replaced
                 FileCheck().check_not("extern_kernels.addmm(").run(code[0])
 
+    def test_replace_mul_zero(self):
+
+        def test(x, y):
+            return (x + (y * 0))
+
+        x = torch.rand([256], device="cuda")
```

Comment:
Hi, may I suggest we replace GPU_TYPE with the hard code `"cuda"` in this test case? This test suit also runs on other GPU, the hard code will fail on XPU.